src/services/share-buttons.service.ts

```diff
@@ -49,6 +49,11 @@ export class ShareButtonsService {
             args = Object.assign({}, args, { via: this.twitterAccount });
         }
 
+        /** check for mobile browser (this won't detect tablets browser) */
+        if (this.window.innerWidth <= 480) {
```

Comment:
@tinesoft this would be my preferred method if it was reliable, but in a similar [plugin](https://hgoebl.github.io/mobile-detect.js/) I read this:

> Warning
> As mentioned later, "User-Agent" based detection is not a reliable solution in most cases, because:
> 
> The rules (regular expressions) are constantly out-dated and incomplete
> You have to update the detection code continuously
> There are other ways to detect how your web application should behave:
> feature detection, e.g. Modernizr
> Media Queries, examples at http://mediaqueri.es/
> Maybe there are some libraries out there (which are probably not free) doing a more reliable job

A more reliable solution would be feature detection, e.g.  check for touch events, however a lot of modern desktop PCs these days have touch screens so even detecting touch support does not guarantee that it is a mobile device or tablet.